workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-cron/build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
 	dependencies {
 		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "latest.release"
-		classpath group: "org.springframework.boot", name: "spring-boot-gradle-plugin", version: "3.4.2"
+		classpath group: "org.springframework.boot", name: "spring-boot-gradle-plugin", version: "2.7.18"
```

Comment:
I recall an issue where someone complained about security concerns with Spring Boot 2, and wanted to update. Is that something we need to be concerned about here?

Reply:
Or does the plugin not have to map to the Spring Boot version?

Reply:
So i tried to update to 3.4.2 and SF bumps it back down. :cry:

Reply:
> I recall an issue where someone complained about security concerns with Spring Boot 2, and wanted to update. Is that something we need to be concerned about here?

That was fixed in the upstream library (com.liferay.client.extension.util.spring.boot3 has fixed the CVE)

Reply:
> Or does the plugin not have to map to the Spring Boot version?

The plugin seems to add the necessary dependencies to the project.